src/bootstrap/src/core/builder.rs

```diff
@@ -917,6 +919,7 @@ impl<'a> Builder<'a> {
             stack: RefCell::new(Vec::new()),
             time_spent_on_dependencies: Cell::new(Duration::new(0, 0)),
             paths,
+            llvm_stamp_hash: OnceLock::new(),
```

Comment:
Adding it right under the `Builder` seems not the correct place. `Build` would be more appropriate, but I would prefer to use the static variable directly right under `fn prebuilt_llvm_config` since that is the only place where this value will be read.

Reply:
I've switched over to a static `OnceLock<String>`.

It's a bit sketchy to have a static storing a value that depends on a particular builder, but if it never sees different values in practice then I guess it's not the end of the world.